Cargo.lock

```diff
@@ -0,0 +1,2083 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+version = 3
+
+[[package]]
+name = "addr2line"
+version = "0.20.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "f4fa78e18c64fce05e902adecd7a5eed15a5e0a3439f7b0e169f0252214865e3"
+dependencies = [
+ "gimli",
+]
+
+[[package]]
+name = "adler"
+version = "1.0.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "f26201604c87b1e01bd3d98f8d5d9a8fcbb815e8cedb41ffccbeb4bf593a35fe"
+
+[[package]]
+name = "ahash"
+version = "0.4.8"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "0453232ace82dee0dd0b4c87a59bd90f7b53b314f3e0f61fe2ee7c8a16482289"
+
+[[package]]
+name = "android-tzdata"
+version = "0.1.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "e999941b234f3131b00bc13c22d06e8c5ff726d1b6318ac7eb276997bbb4fef0"
+
+[[package]]
+name = "android_system_properties"
+version = "0.1.5"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "819e7219dbd41043ac279b19830f2efc897156490d7fd6ea916720117ee66311"
+dependencies = [
+ "libc",
+]
+
+[[package]]
+name = "async-trait"
+version = "0.1.74"
```

Comment:
I just did `cargo +1.55.0 check` and it failed due to this dependency like `cargo-msrv` did. So it'd seem that the expected MSRV of this `Cargo.lock` is `1.56.1` after all?

Reply:
I'm a bit confused, where are you getting 1.55 from?  I think the MSRV of the Cargo.lock I'm proposing is 1.56 (probably 1.56.0, although one should be using 1.56.1 anyway), and 1.56 is what's mentioned in the docs.

Reply:
Sorry didn't see this reply earlier, 1.55 was from the `cargo-msrv` bisect I think, or it was just due to it settling on `1.56.1`, so I tried `1.55.0` to check if it builds, but it failed as mentioned due to `async-trait`.

As you cleared up for me, `tokio` wasn't an issue at `1.33.0` because dev-dependencies were ignored unless running `cargo check --tests`, so the failure was only regarding a release build (_without dev-dependencies `tokio` isn't present in `cargo tree` / `Cargo.lock` either 👍 )